lang_qc/endpoints/config.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2022 Genome Research Ltd.
+#
+# Author: Adam Blanchet <ab59@sanger.ac.uk>
+#
+# This file is part of npg_langqc.
+#
+# npg_langqc is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 3 of the License, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <http://www.gnu.org/licenses/>.
+
+from typing import List
+
+from fastapi import APIRouter
+
+from lang_qc.models.qc_flow_status import QcFlowStatus, QcFlowStatusEnum
+
+router = APIRouter(
+    prefix="/config",
+    tags=["config"],
+)
+
+
+@router.get(
+    "/qc_flow_status",
```

Comment:
I hope we don't need too many separate endpoints here, or first page load will be pretty chatty.